src/requirements/data/majors/foodsci.ts

```diff
@@ -0,0 +1,278 @@
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['FDSC 3950'],
+          ['FDSC 4230', 'FDSC 4250'],
+          ['FDSC 5970'],
+          ['NS 3600', 'PAM 3280', 'ENTOM 4250'],
+          ['PAM 4240', 'DSOC 4080', 'ORIE 2380'],
+          ['BIOMS 4040', 'BIOMS 4090', 'BIOMS 4150', 'BIOMS 4340', 'BIOMG 4390'],
+          [
+            'FDSC 4010',
+            'FDSC 4020',
+            'FDSC 4040',
+            'FDSC 4050',
+            'FDSC 4100',
+            'FDSC 4110',
+            'FDSC 4220',
+            'FDSC 4230',
+            'FDSC 4250',
+            'FDSC 4400',
+            'FDSC 4500',
+            'FDSC 4880',
+            'FDSC 5400',
+          ]
+        ),
+        perSlotMinCount: [1, 1, 1, 1, 1, 1, 1],
+        slotNames: [
+          'FDSC 3950',
+          'FDSC 4230 or FDSC 4250',
+          'FDSC 5970',
+          'NS 3600 or PAM 3280 or ENTOM 4250 (in preferred order)',
+          'PAM 4240 or DSOC 4080 or ORIE 2380 (in preferred order)',
+          'BIOMS 4040 or BIOMS 4090 or BIOMS 4150 or BIOMS 4340 or BIOMG 4390 (in preferred order)',
+          '2 credits of >4000 FDSC electives',
+        ],
+      },
+      Business: {
+        description:
+          'Required FDSC courses, economics course, management course, marketing course, accounting course, finance course, 2 credits of >4000 FDSC electives, and 6 credits of business electives',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
+          ['FDSC 4230'],
+          ['FDSC 5970'],
+          ['ECON 1110', 'ECON 3030', 'PAM 2000'],
+          ['AEM 1200', 'HADM 1810', 'ILRID 1700', 'AEM 2200', 'ILROB 1220'],
+          ['AEM 2400', 'HADM 2410', 'NCC 5530', 'AEM 2420'],
+          ['AEM 2210', 'HADM 2230', 'NCC 5500', 'AEM 2225'],
+          ['AEM 2241', 'NCC 5560', 'HADM 2250', 'AEM 2240'],
+          ['FDSC 4***', 'FDSC 5***', 'FDSC 6***'],
+          [
+            'AEM 3340',
+            'AEM 3440',
+            'AEM 4485',
+            'AEM 3020',
+            'AEM 4420',
+            'AEM 4450',
+            'AEM 3270',
+            'AEM 3230',
+            'AEM 2011',
+            'AEM 2220',
+            'AEM 3249',
+            'AEM 2230',
+            'AEM 2015',
+            'AEM 3120',
+            'AEM 3249',
+            'AEM 4190',
+            'NBA 5070',
+            'AEM 2480',
+            'HADM 3470',
+          ]
+        ),
+        perSlotMinCount: [1, 1, 1, 1, 1, 1, 1, 1, 2],
+        slotNames: [
+          'FDSC 4230',
+          'FDSC 5970',
+          'Economics: ECON 1110 or ECON 3030 or PAM 2000',
+          'Management: AEM 1200 or HADM 1810 or ILRID 1700 or AEM 2200 or ILROB 1220',
+          'Marketing: AEM 2400 or HADM 2410 or NCC 5530 or AEM 2420',
+          'Accounting: AEM 2210 or HADM 2230 or NCC 5500 or AEM 2225',
+          'Finance: AEM 2241 or NCC 5560 or HADM 2250 or AEM 2240',
```

Comment:
Since we already have sensible names for these categories, we don't need to list course code here. The frontend will auto show the eligible courses it's not fulfilled.

Reply:
Thanks, fixed that!